journal/src/main/java/io/camunda/zeebe/journal/file/JournalMetrics.java

```diff
@@ -47,6 +47,14 @@ final class JournalMetrics {
           .labelNames(PARTITION_LABEL)
           .register();
 
+  private static final Histogram JOURNAL_FLUSH_TIME =
```

Comment:
Will measure the complete journal flush time, not the per-segment flush time. I expect they will typically be the same, but I figured it could be interesting in the cases where it's not.